web/styles/portico/navbar.css

```diff
@@ -0,0 +1,570 @@
+.top-menu {
+    position: absolute;
+    top: 0;
+    width: 100%;
+    background: hsla(240, 100%, 10%, 0.4);
+    backdrop-filter: blur(10px);
+    font-family: var(--font-ss3);
+    color: hsl(0, 0%, 100%);
+    z-index: 10;
+    overflow: hidden;
+}
+
+.top-menu__container {
+    /*   1280px + 32px for paddings on the edges */
+    max-width: 1312px;
+    margin: 0 auto;
+    display: flex;
+}
+
+.top-menu__logo {
+    height: 28px;
+    overflow: hidden;
+    opacity: 0.85;
+    padding: 16px;
+    margin-right: 16px;
+    flex-shrink: 0;
+}
+
+.top-menu__items-group-1,
+.top-menu__items-group-2 {
+    display: flex;
+}
+
+.top-menu__items-group-1 {
+    margin-right: 32px;
+}
+
+.top-menu,
+.top-menu-mobile {
+    a,
+    a:hover,
+    a:focus,
+    a:visited {
+        color: hsla(0, 0%, 100%, 0.8);
+    }
+}
+
+.top-menu__item {
+    display: flex;
+    align-items: center;
+    flex-shrink: 0;
+
+    font-family: var(--font-ss3);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    letter-spacing: 0.02em;
+    font-feature-settings: "pnum" on, "lnum" on;
```

Comment:
@terpimost is it intentional that we're using special font settings for menu items in the navbar?

If so, we should probably add a comment for posterity on why, just because it's just 3 elements in this part of the page that have it. (I think these are settings we used for date/times and might not be intentional here?)

Reply:
I guess it could be a mistake. It should be only for digits. But at this point I'm not sure if we have any numbers in the navbar. It could be for lists?

Reply:
I don't think there are ever any digits here; @amanagr I think you can probably remove the couple lines here that are likely from the timestamp CSS.

Reply:
Done.